### DISCUSSION TITLE
Enabling Discussions for Alune

### DISCUSSION THREAD

**Discussion by akshualy** [TeamFightTacticsBots/Alune#56] [Announcements]
## 👋 Welcome!
  We’re now using Discussions as a place to talk about anything that is not an Issue. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Be nice :)

**Comment by ange-loukou** [comment]
hi i do not know if the alunebot is still working but when i folow all steps( i download bluestacks as emulator,i donwload tft app on apk miror then when i finaly come to run alune.exe, the code run and at the end of the code it bring an error with a message saying the app(alune.exe) will close in 10 seconds i need help please 
![error](https://github.com/user-attachments/assets/ae07942e-f912-4ad3-ad35-455d2bc2aa61)

**Reply by akshualy** [reply]
You have to enable ADB in BlueStacks. Here's the Wiki Page on it: https://github.com/TeamFightTacticsBots/Alune/wiki/Emulators#bluestacks

**Reply by ange-loukou** [reply]
thanks you very much,it work

**Comment by 07lule** [comment]
Hello, i got everything running for about 12h, is there a way to auto claim tokens, because i only got 100tokens. Thanks in advance!

**Reply by Kyrluckechuck** [reply]
What do you mean?
I regularly am able to run it for longer than 12h and claim the relevant amount of tokens (typically about ~200)

**Reply by 07lule** [reply]
i can run it longer than 12h thats not the issue. Is there a way to claim these automaticly?
![image](https://github.com/user-attachments/assets/72203e78-aa46-4bac-922e-55ff82cbcbac)

**Reply by akshualy** [reply]
You can claim more than just 100. It just doesn't show more.

**Reply by 07lule** [reply]
i tried but only got 100, is 12h 100tokens?

**Reply by 07lule** [reply]
also got a question how do i setup a surrender

**Reply by akshualy** [reply]
> is 12h 100tokens?

Roughly, 24h is 200-300 tokens, depending on placements.

> how do i setup a surrender

Surrendering is not good for the pass. If you still want to enable it, you can do so in the `alune-output` folder by opening the `config.yaml` and changing the value behind `early_surrender` from `false` to `true`.

**Reply by lolol0l** [reply]
> > Roughly, 24h is 200-300 tokens, depending on placements.

 I get 180 tokens in 24 hours, how can I get 200-300 tokens?
Could you tell me what traits you use and how often to level up and other config?

**Comment by Dastano** [comment]
Any plan to support this Seasonal Sets? Since, from what I understood, you can farm EXP there for the Pass. (the Event that's running for 30 Days rn)

**Reply by akshualy** [reply]
We used to for a past seasonal event, but I just don't have the motivation to update sets this frequently. So from my side, I do not plan to at the moment.
